AdyenCard/Components/Stored Card/StoredCardAlertManager.swift

```diff
@@ -32,11 +33,28 @@ internal final class StoredCardAlertManager: NSObject, UITextFieldDelegate, Loca
         self.cardPublicKeyProvider.clientKey = clientKey
     }
     
-    private let paymentMethod: StoredCardPaymentMethod
-    private let amount: Payment.Amount?
-    
-    internal var cardPublicKeyProvider: AnyCardPublicKeyProvider
-    
+    // MARK: - CVC length
+
+    private var minCharactersCount: Int {
+        guard let brand = CardType(rawValue: paymentMethod.brand) else { return 3 }
+        switch brand {
+        case .americanExpress:
+            return 4
+        default:
+            return 3
+        }
+    }
+
+    private var maxCharactersCount: Int {
+        guard let brand = CardType(rawValue: paymentMethod.brand) else { return 4 }
+        switch brand {
+        case .americanExpress:
+            return 4
+        default:
+            return 3
```

Comment:
max here should be 4 in any case, since its 4-4 for Amex, and 3-4 for any other brand, right?